app/components/activity/index.ts

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+import template from './template.html!text';
+
+export default function(app) {
+  app.component('activity', {
+    bindings: {
+      discussion: '<',
+      document: '<',
+      user: '<',
```

Comment:
This should be renamed to `person` and the provided value should be the id of a person.